other-resources/documentation-and-versions.md

```diff
@@ -0,0 +1,55 @@
+---
+description: The Umbraco Documentation is versioned based on major versions of the Umbraco CMS. Learn more about how that works in this article.
+---
+
+# Documentation and versions
+
+The Umbraco Documentation covers multiple versions of multiple different products. This article explains how we version the documentation as well as how to use it when reading the documentation.
+
+The major version of Umbraco CMS is currently used for versioning the entire Umbraco Documentation site.
+
+{% hint style="info" %}
+The versioning can be disregarded for any and all articles related to Umbraco Cloud and Umbraco Heartcore.
+
+The documentation for Umbraco Cloud and Umbraco Heartcore currently appears with the same versioning as the Umbraco CMS. The plan is to create separate sites for each main Umbraco product; CMS, Cloud, and Heartcore.
+
+Currently, there is no timeline for this change.
+{% endhint %}
+
+## Site-wide versioning
+
+The Umbraco Documentation is versioned on a site-wide basis. This means that when the dropdown is used to change the version, it changes the version of the entire documentation site.
+
+If the article you are currently viewing does not exist in the version you change to, you will be redirected to the documentation landing page.
+
+## Major vs minor versions
+
+The Umbraco Documentation covers all supported versions of Umbraco CMS. A version exists for each supported **major** version of Umbraco CMS.
+
+When a Release Candidate (RC) for a new **minor** version of Umbraco CMS is released, a new version of the documentation will be available. When the minor version is released, the documentation for it will be merged into the related **major** version.
+
+The documentation for each major version will always document the latest version of that major version of Umbraco CMS.
+
+When an RC for a new **major** version of Umbraco CMS is released, a new version of the documentation will be available. When the major version is released, the documentation for it will be the new **default version of the Umbraco Documentation**.
```

Comment:
```suggestion
When a RC for a new **major** version of Umbraco CMS is released, a new version of the documentation will be available. When the major version is released, the documentation for it will be the new **default version of the Umbraco Documentation**.
```

Reply:
I think let's stick with `an` here.
It should be `a` when reading RC as "Release Candidate", but since it's the acronym that written here, it should be `an` 😄 
At least according to Grammarly 😬